tower/Cargo.toml

```
@@ -71,7 +71,7 @@ tower-service = { version = "0.3.1", path = "../tower-service" }
 futures-core = { version = "0.3", optional = true }
 futures-util = { version = "0.3", default-features = false, features = ["alloc"], optional = true }
 hdrhistogram = { version = "7.0", optional = true, default-features = false }
-indexmap = { version = "1.0.2", optional = true }
+indexmap = { version = "2.0", optional = true }
```

You'll need 2.0.2 to satisfy the MSRV check with `-Z minimal-versions`.
```suggestion
indexmap = { version = "2.0.2", optional = true }
```